src/components/Footer/__tests__/Footer.test.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Footer } from '../index';
+
+describe('Footer', () => {
+  const component = <Footer hasTOC={true} />;
+  it('should render the Footer component', async () => {
+    render(component);
+    const footer = await screen.getByRole('contentinfo');
+    expect(footer).toBeInTheDocument();
+  });
+
+  it('should render component that matches snapshot', () => {
+    const footer = render(component);
+    expect(footer).toMatchSnapshot();
```

Comment:
Going off what Scott said for the snapshot, you can access the `container` property to get the html element of what was rendered. Then you can access `outerHTML` to get the HTML as a string

It would look something like this:
```suggestion
    const { container } = render(component);
    expect(container.outerHTML).toMatchSnapshot();
```